### PR TITLE
Add company create form

### LINKED ISSUE #39: Add form for creating companies

### PR DESCRIPTION
**Pull request by lbraun**
Closes #39

### CODE REVIEW

**Comment by lbraun** [comment]
Hey Alaa, check out the form when you get the chance! I hooked it up so it should work for you to create a new company now. I also messed with the css styling a bit, let me know if you don't like any of that and we can revert it.

Now that this is working, we should be able to beef up our tests a lot. I'll add a ticket for that in waffle!

File: app/view/addCompany/addCompany.html [hunk 1]
```diff
@@ -1 +1,26 @@
-<p>This is the partial for adding a new company.</p>
+<div class="row">
+  <div class="col-12 col-lg-8">
+    <h1>Add a new company</h1>
+
+    <form>
```
Comment (alaacs) [inline app/view/addCompany/addCompany.html]: <form ng-model = 'company'>

File: app/view/addCompany/addCompany.html [hunk 2]
```diff
@@ -1 +1,26 @@
...
+      <div class="form-group">
+        <label for="company_name">What's the name of the company you want to add?</label>
+        <input class="form-control" id="company_name" ng-model="company_name" placeholder="Name" required>
```
Comment (alaacs) [inline app/view/addCompany/addCompany.html]: <input class="form-control" id="company_name" ng-model="company.name" placeholder="Name" required>

File: node_app.js [hunk 3]
```diff
@@ -100,6 +113,40 @@ if (false) {
       })
     });
 
+    // * Add a company
+    app.post('/companies', function (req, res) {
+      var name = req.body.name;
+      var wikipedia_name = req.body.wikipedia_name;
+      var industry = req.body.industry;
+
+      var query = format('INSERT INTO companies (name, wikipedia_name, industry) VALUES (\'%s\', \'%s\', \'%s\')', name, wikipedia_name, industry)
```
Comment (alaacs) [inline node_app.js]: you can use 'insert ... returning'
refer to https://stackoverflow.com/a/2944481

File: app/view/addCompany/addCompany.html (hunk 2 above)
Comment (lbraun) [reply app/view/addCompany/addCompany.html]: Something's wrong with your comments dude. Do you see something here? For me it's just a blank comment.

**Comment by lbraun** [comment]
Hey @alaacs, please review again. I found how to use one model in the form (see my last commit). Thanks! 🙇

**Pull request merged**